File: benchmark/comparison/results/blind_ts/18.ts
function flatten(lst: number[][]): number[] {
  const result: number[] = [];
  for (const sub of lst) {
    for (const x of sub) result.push(x);
  }
  return result;
}

function formatList(lst: number[]): string {
  return "[" + lst.join(", ") + "]";
}

console.log(formatList(flatten([[1, 2], [3], [4, 5]])));
console.log(formatList(flatten([[], [1], []])));
console.log(formatList(flatten([])));

File: benchmark/comparison/results/blind_py/18.py
def flatten(lst):
    result = []
    for sub in lst:
        result.extend(sub)
    return result

def fmt(lst):
    return "[" + ", ".join(str(x) for x in lst) + "]"

print(fmt(flatten([[1, 2], [3], [4, 5]])))
print(fmt(flatten([[], [1], []])))
print(fmt(flatten([])))
